File: app/src/main/java/com/teamunemployment/lolanalytics/Utils/RoleUtils.java
package com.teamunemployment.lolanalytics.Utils;

import com.teamunemployment.lolanalytics.data.Statics;

/**
 * Created by Josiah Kendall
 */

public class RoleUtils {

    public String GetRoleName(int role) {
        switch (role) {
            case Statics.ADC:
                return "ADC";
            case Statics.JUNGLE:
                return "Jungle";
            case Statics.MID:
                return "MID";
            case Statics.SUPPORT:
                return "SUPPORT";
            case Statics.TOP:
                return "TOP";
        }
        throw new IllegalStateException("Failed to find correct role. Please set role as role defined" +
                "in Statics class");
    }
}

File: app/src/main/java/com/teamunemployment/lolanalytics/Utils/RoleUtils.kt
package com.teamunemployment.lolanalytics.Utils

import com.teamunemployment.lolanalytics.data.Statics

/**
 * Created by Josiah Kendall
 */

class RoleUtils {

    fun getRoleName(role: Int): String {
        when (role) {
            Statics.ADC -> return "ADC"
            Statics.JUNGLE -> return "Jungle"
            Statics.MID -> return "MID"
            Statics.SUPPORT -> return "SUPPORT"
            Statics.TOP -> return "TOP"
        }
        throw IllegalStateException("Failed to find correct role. Please set role as role defined" + "in Statics class")
    }
}
